Assets/Scripts/Controllers/StatsController.cs

```diff
@@ -111,25 +111,24 @@ public string FormatTime(float t)
             return minutes.ToString ("00") + ":" + seconds.ToString ("00") + ":" + milliseconds.ToString("00");
         }
 
-        public void IncrementFoodEaten()
+        public void IncrementMelonsMunched()
         {
-            foodsEaten++;
+            melonsMunched += 1;
         }
 
-        public void IncrementObjectsSmashed()
+        public void IncrementSabersSlain()
         {
-            objectsSmashed++;
+            sabersSlain += 1;
         }
 
-        public void IncrementBridgesCrossed()
+        public void IncrementTrunksTraversed()
         {
-            bridgesCrossed++;
+            trunksTraversed += 1;
         }
 
-        // TODO Needs to be linked-up.
-        public void IncrementBouldersBumped()
+        public void IncrementFiresFoiled()
         {
-            bouldersBumped++;
+            firesFoiled++;
```

Comment:
Doesn't matter, just wondering why the others changed to +=1 and this one stayed as ++? Were you originally adding more than 1 to some?